jdbc/src/main/java/io/kestra/jdbc/repository/AbstractJdbcDashboardRepository.java

```diff
@@ -167,6 +172,31 @@ public <F extends Enum<F>> ArrayListTotal<Map<String, Object>> generate(String t
         return queryBuilder.fetchData(tenantId, dataChart.getData(), startDate, endDate, pageable);
     }
 
+    @Override
+    public <F extends Enum<F>> ArrayListTotal<Map<String, Object>> generateKPI(String tenantId, DataChartKPI<?, DataFilterKPI<F, ? extends ColumnDescriptor<F>>> dataChart, ZonedDateTime startDate, ZonedDateTime endDate) throws IOException {
```

Comment:
I think you should use a List here not an ArrayListTotal.
ArrayListTotal is for pagination but here you didn't have any